packages/contracts-bedrock/deploy/020-SystemDictatorSteps.ts

```diff
@@ -151,8 +158,15 @@ const deployFn: DeployFunction = async (hre) => {
       console.log(`Setting L1CrossDomainMessenger owner to MSD`)
```

Comment:
The `if (isLiveDeployer)` case hits for devnets so it makes sense not actually send the transactions in the other case because the other case is for upgrading live networks